lib/puppet/trusted_external.rb

```diff
@@ -0,0 +1,13 @@
+# A method for retrieving external trusted facts
+module Puppet::TrustedExternal
+  def retrieve(certname)
+    command = Puppet[:trusted_external_command]
+    return nil if command == "none"
```

Comment:
Can the setting default to `nil` so we don't have to check for `none`?

Reply:
fixed